### PR TITLE
repo: update for v0.2.1.220120

### PR DESCRIPTION
**Pull request by buildbreaker**
Signed-off-by: Alan Chiu <achiu@lyft.com>

For an explanation of how to fill out the fields, please see the relevant section
in [PULL_REQUESTS.md](https://github.com/envoyproxy/envoy/blob/master/PULL_REQUESTS.md)

Description: repo: update for v0.2.1-alpha1
[Optional Fixes #Issue]
[Optional Deprecated:]

### CODE REVIEW

File: docs/root/intro/version_history.rst [hunk 1]
```diff
@@ -1,6 +1,10 @@
 Version history
 ---------------
 
+0.2.1-alpha1 (Jan 6, 2020)
+==========================
+This release contains a ref update for upstream Envoy. No changes to the feature set.
```
Comment (rebello95) [inline docs/root/intro/version_history.rst]: This should have a newline after the `===` and I'd also mention the fixes that @kastiglione merged yesterday

File: docs/root/intro/version_history.rst [hunk 2]
```diff
@@ -1,6 +1,10 @@
 Version history
 ---------------
 
+0.2.1-alpha1 (Jan 6, 2020)
```
Comment (rebello95) [inline docs/root/intro/version_history.rst]: Date is wrong

File: dist/sonatype_nexus_upload.py [hunk 3]
```diff
@@ -287,7 +287,7 @@ def _build_parser():
                             dist/envoy-sources.jar
                             dist/envoy-javadoc.jar
                         """)
-    parser.add_argument("--signed_files", nargs="+", required=True,
+    parser.add_argument("--signed_files", nargs="+", required=False,
```
Comment (rebello95) [inline dist/sonatype_nexus_upload.py]: Why this change?

File: docs/root/intro/version_history.rst (hunk 1 above)
Comment (buildbreaker) [reply docs/root/intro/version_history.rst]: Wasn't that just a build fix?

Comment (rebello95) [reply docs/root/intro/version_history.rst]: yes but I think it's probably worth noting since it could cause problems

File: docs/root/intro/version_history.rst [hunk 4]
```diff
@@ -1,6 +1,13 @@
 Version history
 ---------------
 
+0.2.1-alpha1 (Jan 22, 2020)
+==========================
```
Comment (rebello95) [inline docs/root/intro/version_history.rst]: Docs CI job is failing because this isn't long enough 😬

File: docs/root/development/releasing/releasing.rst [hunk 5]
```diff
@@ -38,3 +38,9 @@ directory, run:
 ``pod trunk push``
 
 Note: This last step is slated to be automated in :issue:`#624 <624>`.
+
+
+Pre-release versioning
+----------------------
```
Comment (rebello95) [inline docs/root/development/releasing/releasing.rst]: I'd make this a larger header (`===`)

File: docs/root/development/releasing/releasing.rst [hunk 6]
```diff
@@ -38,3 +38,9 @@ directory, run:
...
+
+Pre-releases are published on a weekly basis. The versioning scheme we use is ``X.Y.Z.{ddmmyy}``
```
Comment (rebello95) [inline docs/root/development/releasing/releasing.rst]:
```suggestion
Pre-releases are published on a weekly basis. The versioning scheme we use is ``X.Y.Z.{ddmmyy}``.
For example: January 25, 2020: ``0.3.1.250120``.
```
